### ISSUE TITLE
Plan detailed route : Chantilly to Paris

### ISSUE THREAD

**Issue by grd349** [grd349/CycleTrip#11]
We will follow the Avenue Verte all the way to Paris, but once in Paris, we have decisions to make.

Our end point will be Chez Kais.  Do we want to aim for a Champs finish (we did it last time and the traffic was something of a negative).  We could always set an alternative finish point.

What ever we decide, I'd like to make a gpx of the plan and have it in the nav.

**Comment by pxr329** [comment]
I'm guessing anywhere we choose in the centre of Paris is going to be very busy with either traffic or pedestrians? 
Champs does have the tie-ins to TdF, but we could always do something like the Eiffel Tower? or possibly Pont Neuf?

**Comment by grd349** [comment]
sacre-coeur?

**Comment by pxr329** [comment]
Is that the large religious building on the hill we went to? Could make for some good post tour group shots with Paris as a backdrop, and I'm guessing the traffic isn't so bad getting there?

Sounds like a good option to me, guess we'll see if the others have any comments on it.

**Comment by grd349** [comment]
> Sounds like a good option to me, guess we'll see if the others have any comments on it.

I guess the others have no comment :)

**Comment by pxr329** [comment]
OK, I'll knock up a route for comments. 
Just looking at the starting section, for some reason the "official" L'Avenue Verte London-Paris route out of Chantilly does not follow the V32 cycle route out of town, takes a slightly different route and then follows V32 from Gouvieux. Any preference to sticking to L'Avenue, or the V32 out of town?

**Comment by pxr329** [comment]
Here is the route, let me know if you think it needs any tweeking. I followed the L'Avenue Verte route out of Chantilly, from a Google Street view of them both, they looked reasonably similar. 
I've veered off of the route right at the end so we can finish at Sacre-Coeur, there were quite a few different ways up there, so we could change it if needed, looks ok though, the main road in has a cyclelane or bus lane, and I've tried to keep us on the road rather than those stepped sections during the final climb. 
https://ridewithgps.com/routes/30555956
